share/spack/csh/sp_get_generic_arch.csh

```diff
@@ -0,0 +1,16 @@
+#!/bin/csh -f
```

Comment:
@marcmengel this does not contain the expected Spack license (at the beginning of every file) - could you add it?